html/login.html

```diff
@@ -1 +1,99 @@
-로그인 화면입니다.
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/favicon.ico" />
+    <link rel="stylesheet" href="/css/account.css" />
+    <!-- pretendard cdn -->
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <!-- normalize.css CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- fontAwsome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+      crossorigin="anonymous"
+    />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-page">
+      <div class="form">
+        <img
+          src="/images/panda-market-logo.png"
+          alt="판다마켓 이미지"
+          class="form-image"
+          onclick="goHomepage()"
+        />
+        <!-- login form -->
+        <form class="login-form">
+          <!-- email label -->
+          <label for="form-email">
+            이메일
+            <input
+              type="text"
+              name="userEmail"
+              placeholder="codeit@email.com"
+              id="form-email"
+            />
+          </label>
+
+          <!-- password label -->
+          <label for="form-password"
+            >비밀번호
+            <div class="password-box">
+              <input
+                type="password"
+                name="userPassword"
+                placeholder="password"
+                id="form-password"
+              />
+              <i
+                class="fa-regular fa-eye toggle-password"
+                onclick="togglePasswordVisibility('form-password', this)"
+              ></i>
```

Comment:
❗️ 수정요청
상호작용 가능한 요소이니 button 태그로 작성하시는 것을 추천드려요!

```suggestion
              <button onclick="togglePasswordVisibility('form-password', this)" type='button'>
                <i class="fa-regular fa-eye toggle-password"></i>
              </button>
```